bcda/database/connection.go

```diff
@@ -38,3 +38,18 @@ func GetGORMDbConnection() *gorm.DB {
 	}
 	return db
 }
+
+func GetQueueDbConnection() *sql.DB {
```

Comment:
See comment below on `429`.  Maybe we can avoid requiring a connection to the queue db.

Reply:
good concerns, yeah I think putting that page together to explain the different scenarios will give us more of a chance for some good dialogue around this. I just NEED TO FINISH IT.